src/module/item/base/document.ts

```diff
@@ -453,7 +530,11 @@ class ItemPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> extends Item
         htmlOptions: EnrichmentOptionsPF2e = {},
         chatData: RawItemChatData,
     ): Promise<RawItemChatData> {
-        return new ItemChatData({ item: this, data: chatData, htmlOptions }).process();
+        const description = {
+            ...chatData.description,
+            ...(await this.getDescription(htmlOptions)),
+        };
```

Comment:
Both?

Reply:
Its what the ItemChatData process function already did. I'm not sure why, maybe there's other props that chat data might slip in?